_events/2022-07-04-Admin-Training_Freiburg.md

```diff
@@ -0,0 +1,38 @@
+---
+site: freiburg
+tags: [talk, training]
+title: "2022 Galaxy Admin Training"
+starts: 2022-07-04
+ends: 2022-07-08
+organiser:
+  name: Albert Ludwig University of Freiburg
+  email: galaxy@informatik.uni-freiburg.de
+location:
+  city: Freiburg
+  region: Baden-Württemberg
+  country: Germany
+supporters:
+- denbi
+- elixir
+- eurosciencegateway
+- european commission
+---
+
+**[Galaxy Admin Training](https://github.com/galaxyproject/dagobah-training)** will be organized from the **4th to the 8th of July 2022** at the **Albert Ludwig University of Freiburg**, where the **Galaxy Project** is running. 
+The workshop has two parts: [2 day introductory session](https://github.com/galaxyproject/dagobah-training#basic-sessions) and a [2 day advanced topics session](https://github.com/galaxyproject/dagobah-training#advanced-sessions). The registration can be done for either one or both sessions.
+
+This workshop will cover [basic](https://github.com/galaxyproject/dagobah-training#basic-sessions)
+and [advanced topics](https://github.com/galaxyproject/dagobah-training#advanced-sessions) As a prerequisite, it would be useful for you to know to: *set up your own production, high-performance and multi-user Galaxy instance*.  
+The sessions, organized by experienced instructors from the Galaxy Community will be *intensive and hands-on*.
+You will learn how to *install, configure, customize, and extend* your own Galaxy servers. 
+The topics will include: *tool configuration, authentication and user management, using heterogeneous storage and 
+compute services*, as well as many other topics that will enable you to get your own Galaxy server up and running, 
+performing well, and being used by your community.
+
+**Galaxy is an open web based platform for biomedical data integration and analysis. It is deployed at
+large and small organizations around the world and used in a broad range of biomedical research domains,
+and across the tree of life.**
```

Comment:
I think that I would remove this, it is basic information that people interested in the event should know

Reply:
Agreed, we can remove this sentence.

Reply:
I think I removed it - can you still see it?

Reply:
yes, it's still there